apps/aecore/lib/aecore/account/tx/spend_tx.ex

```diff
@@ -115,10 +117,10 @@ defmodule Aecore.Account.Tx.SpendTx do
   before the transaction is executed.
   """
   @spec preprocess_check(
-          ChainState.account(),
+          Chainstate.accounts(),
           tx_type_state(),
           non_neg_integer(),
-          SpendTx.t(),
+          t(),
```

Comment:
I think the full spec is nicer here

Reply:
For Elixir it is custom to write the types without the module name (If we are in the current module of course). @rdimitrovbg and I looked into some of the Elixir documentations and this is how they do it.

Reply:
It is my personal opinion, If I just read some code I can understand way better what is meant, whats your opinion @d-velev @gspasov @cytadela8 ?

Reply:
If this is the way Elixir does it, it should be the way to go.

Reply:
What is your personal opinion?

Reply:
To leave it like this.

Reply:
@d-velev @meivantodorov @cytadela8 (@untra maybe?)